pages/clients/nimbus-portal.md

```diff
@@ -0,0 +1,17 @@
+# Nimbus Portal client
+
+The Nimbus Portal client is an implementation of a [Portal Network client](https://github.com/ethereum/portal-network-specs) written in [Nim](https://nim-lang.org/).
+
+You would use the Nimbus Portal client to interact with the Portal Network. This gives access to Ethereum data with very low CPU and memory requirements and near instant sync.
+
+## Get the Nimbus Portal client
+
+You can download the Nimbus Portal client from the [project Github](https://github.com/status-im/nimbus-eth1/tree/master/portal).
+
+
+## Nimbus Portal client documentation
+
+You can read the [Nimbus Portal guide](https://fluffy.guide)!
```

Comment:
I assume there will be a follow-up URL change.

Reply:
Yes, the work hasn't been done yet to move everything to https://nimbus.guide . Will do follow-up PR when that work gets finished.